### DISCUSSION TITLE
Code of Conduct contains Matrix and Telegram links for Neo Backup

### DISCUSSION THREAD

**Discussion by min7-i** [Akylas/OSS-DocumentScanner#213] [General]
I just wanted to let you know that the [Code of Conduct](https://github.com/Akylas/OSS-DocumentScanner/blob/master/COC.md#enforcement) contains links to a Matrix and Telegram channel for Neo Backup. That's probably just a leftover from the aforementioned project.

**Comment by farfromrefug** [comment]
@min7-i thanks a lot :) indeed should have checked that. Will fix tomorrow.
